core/src/style/style.h

```diff
@@ -99,10 +103,14 @@ class Style {
     /* Toggle on read if true, checks whether the context has been lost on last frame */
     bool glContextLost();
 
+    void setupShaderUniforms(int _lastBoundTex, bool _ctxLost);
+
 private:
 
     /* Whether the context has been lost on last frame */
     bool m_contextLost;
+    std::vector<StyleUniform> m_styleUniforms;
+    std::unordered_map<std::string, std::shared_ptr<Texture>> m_uniformTextures;
```

Comment:
Will it make sense to not have this extra mapping and include "shared_ptr<Texture>" as a UniformValue?